spec/factories/notes.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :note do
+    note { "MyText" }
+    contact { nil }
```

Comment:
Is it possible to reference the Contact factory here, just thinking that contact is a not-null field?  Don't know whether it matters on not, I'm new to this FactoryBot stuff

Reply:
@AlexVPopov Can you help answer Jamies question - I'm not an expert on factory bot either.

Reply:
Yes, the answe is this:

```ruby 
FactoryBot.define do
  factory :note do
    contact

    note { Faker::Lorem.sentence }
  end
end
```